polly-merge.py

```diff
@@ -300,10 +301,10 @@ def merge_after(match):
 
         # dictionary of regex-pattern: command to run on match
         commands = {
-            # command: merge
-            f"^{merge_trigger} merge$": just_merge,
             # command: merge-after <url>
-            f"^{merge_trigger} merge-after (.*)$": merge_after,
+            f"^{merge_trigger} after (.*)$": merge_after,
+            # command: merge
+            f"^{merge_trigger}( merge)?$": just_merge,
```

Comment:
Why this change?

Reply:
:killme: I need to stop being lazy and actually use branches.

This was a local fix that accidentally got added. Y'all broke my merge trigger with your fancy commands. brb